config.json

```diff
@@ -530,6 +530,18 @@
         "recursion"
       ],
       "uuid": "cd573670-54da-4291-ab78-afd5d77fbeac"
+    },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "pig-latin",
+      "topics": [
+        "algorithms",
+        "functions",
+        "strings",
+        "control_flow_if_statements"
+      ],
```

Comment:
This needs an `unlocked_by` value as `core` is `false`

Reply:
Nice catch! It seems I made the same mistake with 'bracket-push' (I made a PR with correction already). It is sad, but I still cannot find a way to automate the generation/updating of 'config.json' and copy/paste errors occur.